src/util/GestureHandler.as

```diff
@@ -133,10 +133,12 @@ public class GestureHandler {
 			}
 		}
 		if (carriedObj && scrollTarget && (getTimer() - scrollStartTime) > SCROLL_MSECS && (scrollXVelocity || scrollYVelocity)) {
-			scrollTarget.contents.x = Math.min(0, Math.max(-scrollTarget.maxScrollH(), scrollTarget.contents.x + scrollXVelocity));
-			scrollTarget.contents.y = Math.min(0, Math.max(-scrollTarget.maxScrollV(), scrollTarget.contents.y + scrollYVelocity));
-			scrollTarget.constrainScroll();
-			scrollTarget.updateScrollbars();
+			if (!scrollTarget is ScriptsPane) {
```

Comment:
Doesn't "ScriptsPane" refer to the whole of the scripts pane?
So scrolling no longer works anywhere in v450...?
See https://scratch.mit.edu/discuss/topic/220438/
(Oops!)